cloudnet-modules/cloudnet-report/src/main/java/de/dytanic/cloudnet/ext/report/CloudNetReportModule.java

```diff
@@ -88,8 +92,15 @@ public void initConfig() {
 
   @ModuleTask(order = 126, event = ModuleLifeCycle.STARTED)
   public void initSavingRecordsDirectory() {
+    SimpleDateFormat dateFormat = new SimpleDateFormat("YYYY-MM-dd");
```

Comment:
Can't we create the format once? And why is the config option for the format not used?